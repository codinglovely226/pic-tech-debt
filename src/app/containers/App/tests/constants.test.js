import {
  BATCH_INSERT_DOCUMENT,
  CHANGE_FORM,
  CLEAR_ERROR,
  DEFAULT_LOCALE,
  DEFAULT_UIMODE,
  DELETE_DOCUMENT,
  DISPLAY_RECORD,
  DISPLAY_TEMPLATE,
  EDITTEMPLATE_UIMODE,
  EXECUTE_SEARCH,
  INIT_SITE_CONFIG,
  INSERT_DOCUMENT,
  LOAD_REPOS,
  LOAD_REPOS_ERROR,
  LOAD_REPOS_SUCCESS,
  LOGIN_REQUEST,
  LOGOUT,
  LOOKUP_MACRO,
  REGISTER_REQUEST,
  REQUEST_ERROR,
  SENDING_REQUEST,
  SET_AUTH,
  SET_TAB,
  SET_UI_MODE_DEFAULT,
  SET_UI_MODE_EDIT_TEMPLATES,
  STORE_KEY,
  SW_UPDATED,
  SYNCING_POUCH,
  UPDATE_ACTION_BAR_SEARCH_RESULTS,
  UPDATE_CASE_LIST_SEARCH_RESULTS,
  UPDATE_DOCUMENT,
  UPDATE_MACRO,
  UPDATE_NEW_RECORD_SEARCH_RESULTS,
  UPDATE_TOXICOLOGY_SOURCE,
} from '../constants';

describe('App action constants', () => {
  it('LOAD_REPOS', () => {
    expect(LOAD_REPOS).toMatchSnapshot();
  });
  it('LOAD_REPOS_SUCCESS', () => {
    expect(LOAD_REPOS_SUCCESS).toMatchSnapshot();
  });
  it('LOAD_REPOS_ERROR', () => {
    expect(LOAD_REPOS_ERROR).toMatchSnapshot();
  });
  it('BATCH_INSERT_DOCUMENT', () => {
    expect(BATCH_INSERT_DOCUMENT).toMatchSnapshot();
  });
  it('CHANGE_FORM', () => {
    expect(CHANGE_FORM).toMatchSnapshot();
  });
  it('CLEAR_ERROR', () => {
    expect(CLEAR_ERROR).toMatchSnapshot();
  });
  it('DEFAULT_LOCALE', () => {
    expect(DEFAULT_LOCALE).toMatchSnapshot();
  });
  it('DEFAULT_UIMODE', () => {
    expect(DEFAULT_UIMODE).toMatchSnapshot();
  });
  it('DELETE_DOCUMENT', () => {
    expect(DELETE_DOCUMENT).toMatchSnapshot();
  });
  it('DISPLAY_RECORD', () => {
    expect(DISPLAY_RECORD).toMatchSnapshot();
  });
  it('DISPLAY_TEMPLATE', () => {
    expect(DISPLAY_TEMPLATE).toMatchSnapshot();
  });
  it('EDITTEMPLATE_UIMODE', () => {
    expect(EDITTEMPLATE_UIMODE).toMatchSnapshot();
  });
  it('EXECUTE_SEARCH', () => {
    expect(EXECUTE_SEARCH).toMatchSnapshot();
  });
  it('INIT_SITE_CONFIG', () => {
    expect(INIT_SITE_CONFIG).toMatchSnapshot();
  });
  it('INSERT_DOCUMENT', () => {
    expect(INSERT_DOCUMENT).toMatchSnapshot();
  });
  it('LOGIN_REQUEST', () => {
    expect(LOGIN_REQUEST).toMatchSnapshot();
  });
  it('LOGOUT', () => {
    expect(LOGOUT).toMatchSnapshot();
  });
  it('LOOKUP_MACRO', () => {
    expect(LOOKUP_MACRO).toMatchSnapshot();
  });
  it('REGISTER_REQUEST', () => {
    expect(REGISTER_REQUEST).toMatchSnapshot();
  });
  it('REQUEST_ERROR', () => {
    expect(REQUEST_ERROR).toMatchSnapshot();
  });
  it('SENDING_REQUEST', () => {
    expect(SENDING_REQUEST).toMatchSnapshot();
  });
  it('SET_AUTH', () => {
    expect(SET_AUTH).toMatchSnapshot();
  });
  it('SET_TAB', () => {
    expect(SET_TAB).toMatchSnapshot();
  });
  it('SET_UI_MODE_DEFAULT', () => {
    expect(SET_UI_MODE_DEFAULT).toMatchSnapshot();
  });
  it('SET_UI_MODE_EDIT_TEMPLATES', () => {
    expect(SET_UI_MODE_EDIT_TEMPLATES).toMatchSnapshot();
  });
  it('STORE_KEY', () => {
    expect(STORE_KEY).toMatchSnapshot();
  });
  it('SW_UPDATED', () => {
    expect(SW_UPDATED).toMatchSnapshot();
  });
  it('SYNCING_POUCH', () => {
    expect(SYNCING_POUCH).toMatchSnapshot();
  });
  it('UPDATE_ACTION_BAR_SEARCH_RESULTS', () => {
    expect(UPDATE_ACTION_BAR_SEARCH_RESULTS).toMatchSnapshot();
  });
  it('UPDATE_CASE_LIST_SEARCH_RESULTS', () => {
    expect(UPDATE_CASE_LIST_SEARCH_RESULTS).toMatchSnapshot();
  });
  it('UPDATE_DOCUMENT', () => {
    expect(UPDATE_DOCUMENT).toMatchSnapshot();
  });
  it('UPDATE_MACRO', () => {
    expect(UPDATE_MACRO).toMatchSnapshot();
  });
  it('UPDATE_TOXICOLOGY_SOURCE', () => {
    expect(UPDATE_TOXICOLOGY_SOURCE).toMatchSnapshot();
  });
  it('UPDATE_NEW_RECORD_SEARCH_RESULTS', () => {
    expect(UPDATE_NEW_RECORD_SEARCH_RESULTS).toMatchSnapshot();
  });
});
