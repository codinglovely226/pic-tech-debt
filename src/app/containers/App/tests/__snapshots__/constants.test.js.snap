exports[`App action constants BATCH_INSERT_DOCUMENT 1`] = `"pic/App/BATCH_INSERT_DOCUMENT"`;

exports[`App action constants CHANGE_FORM 1`] = `"pic/App/CHANGE_FORM"`;

exports[`App action constants CLEAR_ERROR 1`] = `"pic/App/CLEAR_ERROR"`;

exports[`App action constants DEFAULT_LOCALE 1`] = `"en"`;

exports[`App action constants DEFAULT_UIMODE 1`] = `"default"`;

exports[`App action constants DELETE_DOCUMENT 1`] = `"pic/App/DELETE_DOCUMENT"`;

exports[`App action constants DISPLAY_RECORD 1`] = `"pic/App/DISPLAY_RECORD"`;

exports[`App action constants DISPLAY_TEMPLATE 1`] = `"pic/App/DISPLAY_TEMPLATE"`;

exports[`App action constants EDITTEMPLATE_UIMODE 1`] = `"editTemplate"`;

exports[`App action constants EXECUTE_SEARCH 1`] = `"pic/App/EXECUTE_SEARCH"`;

exports[`App action constants INIT_SITE_CONFIG 1`] = `"pic/App/INIT_SITE_CONFIG"`;

exports[`App action constants INSERT_DOCUMENT 1`] = `"pic/App/INSERT_DOCUMENT"`;

exports[`App action constants LOAD_REPOS 1`] = `"boilerplate/App/LOAD_REPOS"`;

exports[`App action constants LOAD_REPOS_ERROR 1`] = `"boilerplate/App/LOAD_REPOS_ERROR"`;

exports[`App action constants LOAD_REPOS_SUCCESS 1`] = `"boilerplate/App/LOAD_REPOS_SUCCESS"`;

exports[`App action constants LOGIN_REQUEST 1`] = `"pic/App/LOGIN_REQUEST"`;

exports[`App action constants LOGOUT 1`] = `"pic/App/LOGOUT"`;

exports[`App action constants LOOKUP_MACRO 1`] = `"pic/App/LOOKUP_MACRO"`;

exports[`App action constants REGISTER_REQUEST 1`] = `"pic/App/REGISTER_REQUEST"`;

exports[`App action constants REQUEST_ERROR 1`] = `"pic/App/REQUEST_ERROR"`;

exports[`App action constants SENDING_REQUEST 1`] = `"pic/App/SENDING_REQUEST"`;

exports[`App action constants SET_AUTH 1`] = `"pic/App/SET_AUTH"`;

exports[`App action constants SET_TAB 1`] = `"pic/App/SET_TAB"`;

exports[`App action constants SET_UI_MODE_DEFAULT 1`] = `"pic/App/SET_UI_MODE_DEFAULT"`;

exports[`App action constants SET_UI_MODE_EDIT_TEMPLATES 1`] = `"pic/App/SET_UI_MODE_EDIT_TEMPLATES"`;

exports[`App action constants STORE_KEY 1`] = `"pic/App/STORE_KEY"`;

exports[`App action constants SW_UPDATED 1`] = `"pic/App/SW_UPDATED"`;

exports[`App action constants SYNCING_POUCH 1`] = `"pic/App/SYNCING_POUCH"`;

exports[`App action constants UPDATE_ACTION_BAR_SEARCH_RESULTS 1`] = `"pic/App/UPDATE_ACTION_BAR_SEARCH_RESULTS"`;

exports[`App action constants UPDATE_CASE_LIST_SEARCH_RESULTS 1`] = `"pic/App/UPDATE_CASE_LIST_SEARCH_RESULTS"`;

exports[`App action constants UPDATE_DOCUMENT 1`] = `"pic/App/UPDATE_DOCUMENT"`;

exports[`App action constants UPDATE_MACRO 1`] = `"pic/App/UPDATE_MACRO"`;

exports[`App action constants UPDATE_NEW_RECORD_SEARCH_RESULTS 1`] = `"pic/App/UPDATE_NEW_RECORD_SEARCH_RESULTS"`;

exports[`App action constants UPDATE_TOXICOLOGY_SOURCE 1`] = `"pic/App/TOXICOLOGY_SOURCE"`;
